src/services/rfqm_service.ts

```diff
@@ -76,6 +76,25 @@ export interface BaseRfqmQuoteResponse {
     sellTokenAddress: string;
 }
 
+interface OrderStatusPending {
+    status: 'pending';
+}
+
+interface OrderStatusSubmitted {
+    status: 'submitted';
+    transactions: { hash: string; timestamp: number /* unix ms */ }[];
```

Comment:
I think this is okay (I get the logic) but mildly worried about a different properties appearing on the response schema (transaction vs. transactions)

Reply:
@PirosB3 made it sound like this is an internal only type thing. If that's the case any TS users can just import the type, correct??

Reply:
response type has been simplified to just `transactions`